charmcraft/commands/pack.py

```diff
@@ -100,6 +100,11 @@ def fill_parser(self, parser):
             type=pathlib.Path,
             help="Dump measurements to the specified file",
         )
+        parser.add_argument(
+            "--disable-shared-cache",
+            action="store_true",
```

Comment:
I thought we said not to do this now, in any case, if we are keeping it,  we should also use `dest=shared_cache_enabled` and make the action store False, and the default to be True to avoid negative positive checks which play tricks when reading code.

@lengau ?

Reply:
Definitely agree that we should prefer a `store_false` here if we're keeping the argument. 

I'm okay to drop the ability to disable the shared cache too — that originally came through my intention to start with it being an optional enable that we'd eventually turn on by default.

Reply:
OK, then I remove that part.